docs/getting-started.md

```diff
@@ -34,221 +37,359 @@ The high level data flow is as follows:
+### 0.4 Getting Started Overview
+Below we outline the required steps to build a Subgraph from scratch, that will be serving up queries from a GraphQL endpoint. There are three major steps:
 
-The subgraph is defined as a YAML file called a *subgraph manifest*. See [here](https://github.com/graphprotocol/graph-cli/blob/master/examples/example-event-handler/subgraph.yaml) for an example, or [here](subgraph-manifest.md) for the full subgraph manifest specification. It comprises a schema, source data and mappings which are used to deploy your endpoint.
+1. Define The Subgraph
 
-The subgraph manifest is typically placed into a *subgraph* directory. We'll also add a `package.json` and `tsconfig.json` in the directory, to take advantage of our Javascript-based build toolchain using the Graph CLI.
+    1. [Create a subgraph project and manifest](#3-defining-your-subgraph)
+    
+    2. [Define a GraphQL schema](#31-defining-your-graphql-schema)
+    
+    3. [Define your source data](#32-defining-your-source-data)
+    4. [Generate types for your mapping with the Graph-CLI](#33-generate-types-for-your-mapping-with-the-graph-cli)
+    5. [Write your mappings](#34-write-your-mappings)
+2. Build and Deploy The Subgraph
+    1. [Build and deploy your mappings to IPFS](#41-build-and-deploy-your-mappings-to-ipfs)
+    
+    1. [Deploy your subgraph to your local Graph Node](#42-deploy-your-subgraph-to-your-local-graph-node)
+3. Query The Subgraph
+    1. [Query your newly deployed GraphQL API](#5-query-your-local-graph-node)
 
-Before proceeding, follow the instructions in the [Graph CLI Readme](https://github.com/graphprotocol/graph-cli/) for setting up your subgraph directory.
-
-### 3.1 Defining your GraphQL schema
-GraphQL schemas are defined using the GraphQL interface definition language (IDL). If you've never written a GraphQL schema, we recommend checking out a [quick primer](https://graphql.org/learn/schema/#type-language) on the GraphQL type system.
+Now let's dig in!
 
-With The Graph, you don't have to define your own top-level `Query` type, you simply define entity types, and Graph Node will generate top level fields for querying single instances and collections of that entity type. Each type that should be an entity is required to be annotated with an `@entity` directive.
-
-##### Example
-Define a simple Token entity type:
-```graphql
-type Token @entity {
-  id: ID!
-  name: String!
-  minted: Int!
-}
-```
-
-Later, when you've deployed your subgraph with this entity, you'll be able to query for Tokens:
-
-```graphql
-query {
-  token(id: "123") {
-    name
-    minted
-  }
-}
-```
+## 3 Defining your Subgraph 
+In The Graph, we refer to your project's GraphQL endpoint as a *subgraph*, because once deployed to the decentralized network, it will be just one subset of a global GraphQL endpoint.
```

Comment:
It will not "be just one subset of a global GraphQL endpoint". That's sounds insignificant.

`It will form a part of a global graph of blockchain data.`?